src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: Onramps style guide
+contentType: page
+template: basicDoc
+topics:
+  - onramps
+---
+
+import apmOnrampsIcon from 'images/apm_icon_onramps-icon.webp'
+
+import apmActiveEmptyState from 'images/apm_screenshot-crop_active-empty-state.webp'
+
+Onramps content is information that appears our UI when a user clicks a <img
+    title=""
+    alt=""
+    src={apmOnrampsIcon}
+/>
+ icon. This content is meant to be truncated and contextualized to where the user is and what they need to know to perform a task.
+
+## How should I name the titles of onramps content?
+
+### FAQ structure
+
+When applicable, use an FAQ format where the headers are in question format.
+
+Example:
+
+**Can I get notifications for issues?**
+You can get notifications through email, Slack, Jira, and more. To learn how to set up notifications, see [Introduction to alert notifications](/docs/alerts-applied-intelligence/notifications/intro-notifications/).
+
+### Regular headers
+
+If FAQ format is not applicable, use our standard headers for titles.
+
+Example:
+
+**Forward logs to New Relic**
+Forwarding logs to New Relic allows you to have all logs in one place.
```

Comment:
@zuluecho9 Do you have any examples of onramps that's not in a question format? I just added this as placeholder.